packages/roosterjs-editor-api/lib/format/insertTable.ts

```diff
@@ -3,6 +3,15 @@ import execFormatWithUndo from './execFormatWithUndo';
 
 const ZERO_WIDTH_SPACE = '&#8203;';
 
+/**
+ * The table format
+ * @param cellSpacing (Optional) The cellSpacing style for the HTML table element
```

Comment:
Don't add comment for interface member here, but add comment to each member, e.g.

/**
 The cellPadding style for the HTML table element
*/
cellPadding?: string;
...

Reply:
Changed.